lib/realtime/tenants/scheduled_message_cleanup.ex

```diff
@@ -0,0 +1,95 @@
+defmodule Realtime.Tenants.ScheduledMessageCleanup do
+  @moduledoc """
+  Scheduled tasks for the Tenants.
+  """
+
+  use GenServer
+  require Logger
+
+  import Ecto.Query
+  import Realtime.Helpers, only: [log_error: 2]
+
+  alias Realtime.Api.Tenant
+  alias Realtime.Database
+  alias Realtime.Messages
+  alias Realtime.Nodes
+  alias Realtime.Repo
+
+  defstruct timer: nil, region: nil, chunks: nil
+
+  def start_link(_args) do
+    timer = Application.get_env(:realtime, :schedule_clean, :timer.hours(5))
```

Comment:
A small hint to avoid running all selects across the cluster at the same time
```suggestion
    timer = Application.get_env(:realtime, :schedule_clean, Enum.random(250..350) * 1000)
```

Reply:
every new send_after can be invoked with a random timeout

Reply:
good one 👍  will adapt slightly